device/vxlan/unicast.yaml

```diff
@@ -1,45 +1,75 @@
 components:
   schemas:
-    Vxlan.Unicast:
+    Vxlan.V4Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V4Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.V6Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V6Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.Tunnel.DestinationIPMode.Unicast.VtepBase:
       description: >-
-        Configuration of Unicast parameters for VxLAN.
+        DestinationIPMode Unicast VTEP Base
       type: object
       properties:
-        remote_vtep_address:
-          description: >-
-            Remote VXLAN Tunnel End Point address
-          $ref: '#/components/schemas/Remote.Vtep.Address'
-        suppress_arp:
-          description: >-
-            Supress ARP 
-          type: boolean
-          default: false
-        remote_vm_mac:
-          description: >-
-            Remote VM MAC addresses
+        arp_suppression_cache:
+          description: >- 
+            VM arps are supressed by resolving against the static locally configured db.
+            If arp entry is not found in the local static db,arp requests are send.
```

Comment:
Re-adding my comment which I dont see resolved or updated : ""When VMs behind this VXLAN tunnel initiate an ARP Request for a remote VM, first a lookup is done in the local arp_suppression_cache(s) across all remote VTEPs based on the remote IPv4 address in the ARP Request. If an entry is found, the VXLAN endpoint suppresses the ARP Requests by creating corresponding ARP Response and sending it back to the VM. If no entry is found, the ARP request is encapsulated with VXLAN header and sent to all the Remote VTEPs" Please modify as needed but above explanation for an obscurish internal feature specific to Unicast ( seems to be no clear reason why not supported for Multicast and why IPv6 cache is not present) is not enough. Since this is not RFC specific, we need a more detailed explanation for this for end-user. For attributes coming from RFC, I am not so worried since VXLAN is an advanced feature and end-user would need to have some idea about the internal of VXLAN to set up a test. But this arp suppression thing seems very much a value-add feature which is not defined in the RFC.

Reply:
yes, I agree..Have added more description for end-user understanding

For `Multicast` & `IPv6 cache`, we can add these properties later based on user's requirement as these will be additional properties and will not break the existing.